chord_metadata_service/restapi/utils.py

```diff
@@ -3,6 +3,7 @@
 
 
 COMPUTED_PROPERTY_PREFIX = "__"
+MAPPING_SEPARATOR = "/"
```

Comment:
duplicate declarations; consolidate to one location.

Reply:
was not even used :sweat_smile: